custom/icds_reports/utils/aggregation_helpers/monolith/aww_incentive.py

```diff
@@ -68,11 +68,10 @@ def aggregation_query(self):
             awcm.is_launched = 'yes',
             round(sum(ccsm.expected_visits)),
             awcm.awc_days_open >= 21,
-            CASE 
-                WHEN round(sum(ccsm.expected_visits)) = 0 AND awcm.wer_eligible = 0 THEN true
-                ELSE (awcm.wer_weighed / GREATEST(awcm.wer_eligible, 1)) >= 0.6 AND 
-                     (sum(ccsm.valid_visits) / GREATEST(round(sum(ccsm.expected_visits)), 1)) >= 0.6
-            END
+            (ROUND(awcm.wer_weighed / GREATEST(awcm.wer_eligible, 1)::NUMERIC, 4) >= 0.6 
+                    OR COALESCE(awcm.wer_eligible, 0) = 0)
+                AND (ROUND(sum(ccsm.valid_visits) / GREATEST(round(sum(ccsm.expected_visits)), 1)::NUMERIC, 0) >= 0.6 
```

Comment:
looks like this could result in a division by `0`

Reply:
the `GREATEST` prevents that. the `0` here is the round precision (which should be 4 and is an error, but not that one)

Reply:
right, got it, too many `()`